### ISSUE TITLE
interactive use not working?

### ISSUE THREAD

**Issue by jwnki** [rougier/persid#5]
Hi, thanks a lot for this package. Maybe I'm missing something but I am unable to use it in the interactive mode. I.e. when I do `M-x persid-from-bibtex <RET>` I see "Identifier: " prompt in the minibuffer. So I type: `doi:10.1080/02699206.2019.1673486` and press enter, which outputs "Contacting host: api.crossref.org:443" in the echo area but nothing more happens. 
It does work when I evaluate `(persid-from-bibtex "doi:10.1080/02699206.2019.1673486")`.

**Comment by plants-mom** [comment]
Sorry, maybe I wasn't specific enough. I did try what you are suggesting and it works. I.e. evaluating `(insert (persid-bibtex-from "doi:10.1080/02699206.2019.1673486"))` inserts a formatted bibtex entry into the current buffer.  
Doing `M-: (persid-bibtex-from "doi:10.1080/02699206.2019.1673486")` also works, i.e. it outputs a bib entry into *Messages* buffer. 
However, if I do `M-x <RET> persid-bibtex-from` and then provide an identifier at the minibuffer prompt it doesn't work. It outputs `Contacting host: api.crossref.org:443` into the *Messages* buffer but the bib entry is not there, and not anywhere else as far as I can tell.

**Comment by rougier** [comment]
You mean you would prefer the command to actually insert the bibtex in the current buffer (or to display a message with the bibtex)?

**Comment by rougier** [comment]
No, I get the same surprise recently actually. The interactive part make it confusing and may inserting at point would be more explicit or we rename the function, e.g. `persid-insert-bibtex-from-doi`.
